styles.css

```diff
@@ -137,16 +131,338 @@ p {
 }
 
 ul {
-  position: absolute;
   align-items: center;
   padding: initial;
   display: inline-flex;
   flex-direction: row;
   flex-wrap: nowrap;
   justify-content: space-around;
   list-style: none;
+  margin: 10px;
 }
 
-body {
-  margin: 1px;
+.proyects {
+  background: #fff;
+  flex-direction: column;
+  align-items: center;
+  display: flex;
+  height: 3455px;
+  left: 1px;
+  top: 682px;
+}
+
+.imgprincipal {
+  padding: 24px;
+}
+
+.fontstitles {
+  margin-left: 24px;
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border-bottom: 1px groove black;
+  flex-wrap: nowrap;
+  margin-bottom: 80px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1px;
+  gap: 24px;
+  width: 375px;
+  height: 3028px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  border: 1px solid black;
+}
+
+.head1 {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.letter1 {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.letter2 {
+  width: 327px;
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
```

Comment:
- [OPTIONAL] Please follow the template provided by the Figma related to the design of these texts. Use the correct spaces defining margins or paddings. Check for other parts of your code with the same issue. As not requested by the previous reviewer, I consider that optional.

Reply:
ok i edit it and change it